index.html

```diff
@@ -36,14 +36,15 @@ <h1 class="text-lg text-gray-50">SkyGuide</h1>
       <menu>
         <div
           id="city-list"
-          class="z-3 fixed bottom-0 right-0 h-3/4 translate-x-64 flex-row rounded-l-2xl bg-gradient-to-r from-blue-500 via-blue-400 to-blue-300 p-6 pt-12 md:left-0 md:right-auto md:-translate-x-64 md:rounded-l-none md:rounded-r-2xl"
+          class="z-3 fixed bottom-0 right-0 h-3/4 translate-x-64 flex-row rounded-l-2xl bg-gradient-to-r from-blue-500 via-blue-400 to-blue-300 pt-12 md:left-0 md:right-auto md:-translate-x-64 md:rounded-l-none md:rounded-r-2xl"
```

Comment:
I know it was a small fix, but just make sure you are only doing local storage stuff in this branch.

Reply:
Unless you need it for local storage 🤷🏻‍♂️

Reply:
Oh yeah, this is styling related to the LocaStorage. Thank you!